src/commands/gate/utils.ts

```diff
@@ -1,18 +1,15 @@
+import {datadogSite} from '../../helpers/api'
 import {getCommonAppBaseURL} from '../../helpers/app'
 
 export const getBaseUrl = () => {
-  const site = process.env.DD_SITE || 'datadoghq.com'
+  const site = process.env.DATADOG_SITE || process.env.DD_SITE || 'datadoghq.com'
```

Comment:
I'm guessing this one should also be changed to use the imported value?

Reply:
Oh I missed this, I orginally was going to target the git-metadata command as this was the command that inspired me to make the code change. I wanted to be a bit more ambitious after. 

I will fix this.